src/go/rpk/go.sum

```diff
@@ -286,6 +286,8 @@ github.com/tmc/grpc-websocket-proxy v0.0.0-20190109142713-0ad062ec5ee5 h1:LnC5Kc
 github.com/tmc/grpc-websocket-proxy v0.0.0-20190109142713-0ad062ec5ee5/go.mod h1:ncp9v5uamzpCO7NfCPTXjqaC+bZgJeR0sMTm6dMHP7U=
 github.com/ugorji/go v1.1.4 h1:j4s+tAvLfL3bZyefP2SEWmhBzmuIlH/eqNuPdFPgngw=
 github.com/ugorji/go v1.1.4/go.mod h1:uQMGLiO92mf5W77hV/PUCpI3pbzQx3CRekS0kk+RGrc=
+github.com/vectorizedio/redpanda v20.12.9+incompatible h1:cf45bkRRzQ35+AXjeObwo0vkJu0tdjZE3A53UyY9YfQ=
+github.com/vectorizedio/redpanda/src/go/rpk v0.0.0-20210111154524-d4679492ff6b h1:l9mNnQruPJkJcQjCJkODLYJcp1+0yKCdw4ZJ+pg8SCg=
```

Comment:
why is go sum referencing this package? 🤔 is this after 'go mod tidy'?

Reply:
I didn't run `go mod tidy` before push.

Reply:
Strange. The `go mod tidy` didn't remove the mentioned 2 lines. I removed them manually.